b2sdk/api_config.py

```diff
@@ -8,30 +8,33 @@
 #
 ######################################################################
 
+from typing import Optional, Callable, Type
 import requests
 
-from typing import Optional
-
-from types import ModuleType
+if False:
+    from .raw_api import AbstractRawApi
```

Comment:
?

Reply:
The symbol is needed for type annotations but the module cannot be imported because that would cause circular dependencies

Reply:
I've fixed the circular imports - they were caused by the tests in raw_api.